src/dbos.ts

```diff
@@ -407,6 +410,17 @@ export class DBOS {
     recordDBOSLaunch();
   }
 
+  static internalQueue: WorkflowQueue | undefined = undefined;
+
+  static createInternalQueue() {
+    if (DBOS.internalQueue !== undefined) {
+      DBOS.logger.debug('Internal queue already created!');
+      return;
+    }
+    DBOS.logger.debug('Creating internal queue');
+    DBOS.internalQueue = new WorkflowQueue(INTERNAL_QUEUE_NAME);
+  }
+
```

Comment:
Any methods on the `DBOS` class is external facing. Let's move internal queue to `DBOSExecutor` class.